CSharpMath.Rendering/Settings.cs

```diff
@@ -1,20 +1,24 @@
 namespace CSharpMath {
   using System.Collections.Generic;
-  using CSharpMath.Structures;
+  using Structures;
   public static class Settings {
     public static Rendering.BackEnd.Typefaces GlobalTypefaces =>
       Rendering.BackEnd.Fonts.GlobalTypefaces;
     public static BiDictionary<string, System.Drawing.Color> PredefinedColors =>
       Atom.LaTeXSettings.PredefinedColors;
-    public static AliasDictionary<string, Atom.Boundary> PredefinedLaTeXBoundaryDelimiters =>
+    public static LaTeXCommandDictionary<Atom.Boundary> PredefinedLaTeXBoundaryDelimiters =>
       Atom.LaTeXSettings.BoundaryDelimiters;
-    public static AliasDictionary<string, Atom.FontStyle> PredefinedLaTeXFontStyles =>
+    public static BiDictionary<string, Atom.FontStyle> PredefinedLaTeXFontStyles =>
       Atom.LaTeXSettings.FontStyles;
-    public static AliasDictionary<string, Atom.MathAtom> PredefinedLaTeXCommands =>
+    public static LaTeXCommandDictionary<System.Func
```

Comment:
Why is the type argument here so complex? Is there a need to take an Atom.LaTeXParser as an input in the System.Func? Isn't the LaTeXParser "globally" known? What is the char? Why is the output a Result with two type arguments and what does the name Return mean in second argument case (is that the error case?). Please consider review comments seriously, i.e. think about them and if there is a possibility of simplifying this. Once simplified this will need to be documented.

Reply:
>Why is the type argument here so complex?

A parser itself is complex.
>Is there a need to take an Atom.LaTeXParser as an input in the System.Func?

Yes. It stores the current parsing status and provides methods for reading arguments - you can deduce this if you looked at the code below.
>Isn't the LaTeXParser "globally" known?

No. LaTeXParser is not a singleton - that would mean locking and not being able to parallelize.
>What is the char?

It is called `stopChar`, seen in every lambda below. It is the char to stop at and return, e.g. stopChar is `}` after `{` is encountered.
>Why is the output a Result with two type arguments and what does the name Return mean in second argument case (is that the error case?).

No. There are commands that return `MathAtom`s (e.g. `\frac`) and `MathList`s (e.g. `\mathrm` - the style is stored in each atom).
>Please consider review comments seriously, i.e. think about them and if there is a possibility of simplifying this.

You are saying this while you question code refactored from the original code but this PR simply moves code around. You cannot expect people who refactor code to know everything about the original code - the code still works as intended if tests pass. The tone of "Please consider review comments seriously" means that you think your comments are not being taken seriously at the time of writing your comment. Is this really the case? Are discussions not welcome?
>Once simplified this will need to be documented.

However, your "simplifications" are still under question.